build-system/babel-config/pre-closure-config.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS-IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const {devDependencies} = require('./dev-dependencies');
+const {replacePlugin} = require('./replace-plugin');
+
+/**
+ * Gets the config for pre-closure babel transforms run during `gulp dist`.
+ *
+ * @return {!Object}
+ */
+function getPreClosureConfig() {
+  const isCheckTypes = argv._.includes('check-types');
+  const filterImportsPlugin = [
+    'filter-imports',
+    {
+      imports: {
+        // Imports removed for all ESM builds.
+        './polyfills/document-contains': ['installDocContains'],
+        './polyfills/domtokenlist': ['installDOMTokenList'],
+        './polyfills/fetch': ['installFetch'],
+        './polyfills/math-sign': ['installMathSign'],
+        './polyfills/object-assign': ['installObjectAssign'],
+        './polyfills/object-values': ['installObjectValues'],
+        './polyfills/promise': ['installPromise'],
+        './polyfills/array-includes': ['installArrayIncludes'],
+        '../third_party/css-escape/css-escape': ['cssEscape'],
```

Comment:
A small heads up about https://github.com/ampproject/amphtml/pull/27629

This PR added another item to the `imports` object literal here.
`'./ie-media-bug': ['ieMediaCheckAndFix'],`

Reply:
Appreciate the heads up. Included.